repose-aggregator/build.gradle

```diff
@@ -7,12 +7,26 @@ dependencies {
     asciidoctor 'org.asciidoctor:asciidoctorj-diagram'
 }
 
+task copyXsds(type: Copy) {
+    // Copy the Repose Core configuration schemas.
+    from("${project.projectDir}/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd")
+    from("${project.projectDir}/core/repose-core-api/src/main/resources/META-INF/schema/metrics/metrics.xsd")
+    from("${project.projectDir}/core/repose-core-api/src/main/resources/META-INF/schema/system-model/system-model.xsd")
```

Comment:
These changes don't need to be made now, but I think it's worth pointing out that the `metrics.xsd` schema isn't being referenced by any page, and we'll eventually need to add:
`core/repose-core-api/src/main/resources/META-INF/schema/response-messaging/response-messaging.xsd`